README.md

```diff
@@ -1,48 +1,73 @@
-Role Name
+Pacifica
 =========
 
-A brief description of the role goes here.
+This Ansible role defines configuration for deploying Pacifica services in an
+Ansible managed infrastructure.
 
 Requirements
 ------------
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the boto package is required.
+There are no requirements for this role currently. However, the Pacifica
+services do depend on other services depending on configuration. So, please
+be aware if you configure a service to use PostgreSQL you should configure
+PostgreSQL prior to includeing this role.
```

Comment:
Consider rewrite here:
-PostgreSQL prior to includeing this role.
+PostgreSQL prior to including this role as part of your deployment.

Reply:
Done